composites/OnboardingWizard/onboarding-wizard.scss

```diff
@@ -316,3 +315,7 @@ body {
     box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.1);
   }
 }
+
+.#{$prefix}-content-container {
```

Comment:
Please add spaces within brackets.

Worth noting the whole file is using spaces for indentation, instead it should use tabs but this is out of the scope of this PR.